LICENSE.txt

```diff
@@ -35,7 +35,7 @@ Mozilla Public License Version 2.0
     means any form of the work other than Source Code Form.
 
 1.7. "Larger Work"
-    means a work that combines Covered Software with other material, in 
+    means a work that combines Covered Software with other material, in
```

Comment:
"work" is not quantifiable in most English dialects. I would prefer to remove "a", since you have already edited this line.